components/api-resource-mgt/org.wso2.carbon.identity.api.resource.collection.mgt/src/main/java/org/wso2/carbon/identity/api/resource/collection/mgt/constant/APIResourceCollectionManagementConstants.java

```diff
@@ -54,6 +54,11 @@ public static class APIResourceCollectionConfigBuilderConstants {
         public static final String TYPE = "type";
         public static final String READ = "Read";
         public static final String FEATURE = "Feature";
+        public static final String VERSION = "version";
+        public static final String VIEW_FEATURE_SCOPE_PREFIX = "_view";
```

Comment:
Shall we rename the variable to VIEW_FEATURE_SCOPE_SUFFIX?

Reply:
EDIT_FEATURE_SCOPE_SUFFIX

Reply:
fixed